.github/workflows/pr-actions.yml

```
@@ -1,12 +1,17 @@
     types: [created, edited]
-jobs:
+  
+jobs: 
   publish:
+    if: github.event.issue.pull_request && contains(github.event.comment.body, '/publish')
     name: Publish PR packages
     runs-on: ubuntu-latest
-    if: github.event.issue.pull_request && contains(github.event.comment.body, '/publish')
+    permissions:      
+      contents: read
+      metadata: read
```

actionlint (1.7.8)

unknown permission scope "metadata". all available permission scopes are "actions", "attestations", "checks", "contents", "deployments", "discussions", "id-token", "issues", "models", "packages", "pages", "pull-requests", "repository-projects", "security-events", "statuses"

(permissions)